consensus/consensus_v2.go

```diff
@@ -612,20 +638,22 @@ func (consensus *Consensus) onPrepared(msg *msg_pb.Message) {
 	blockNumBytes := make([]byte, 8)
 	binary.LittleEndian.PutUint64(blockNumBytes, consensus.blockNum)
 	commitPayload := append(blockNumBytes, consensus.blockHash[:]...)
-	msgToSend := consensus.constructCommitMessage(commitPayload)
+	for i, key := range consensus.priKey.PrivateKey {
+		msgToSend := consensus.constructCommitMessage(commitPayload, consensus.PubKey.PublicKey[i], key)
 
-	// TODO: genesis account node delay for 1 second, this is a temp fix for allows FN nodes to earning reward
-	if consensus.delayCommit > 0 {
-		time.Sleep(consensus.delayCommit)
-	}
+		// TODO: genesis account node delay for 1 second, this is a temp fix for allows FN nodes to earning reward
+		if consensus.delayCommit > 0 {
```

Comment:
move it outside, so that the total delay time is 2s instead 2 times number of keys

Reply:
okay